src/ipahealthcheck/core/output.py

```diff
@@ -16,6 +17,26 @@ class OutputRegistry(Registry):
 output_registry = OutputRegistry()
 
 
+class Int:
+    def __init__(self, minimum=0, maximum=0):
```

Comment:
The default value maximum=0 currently does not cause any issue because the only use of this class is setting maximum=32. But for future uses, creating an Int() without parameters would result in only the 0 value allowed. By default it would make more sense to have default maximum=a non-null value.
A check to ensure that minimum < maximum could be added.

Reply:
Yeah I guess. I like to give folks a lot of rope to hang themselves with but I can set a reasonable minimum, I just don't know what this might be used for in the future so whatever I choose will be wrong. But it doesn't have to be a gotcha  I guess.